### PR TITLE
Added Bill Lee as responsible for B0 preshower

### LINKED ISSUE #21: Adding the improved preshower for B0

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
Bill Lee @billlee77 will be responible for the B0 Preshower implementation.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [X] New feature (issue epic #91)
- [X] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [X] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No

### Does this PR change default behavior?
No

### CODE REVIEW

File: ip6/far_forward/B0_preshower.xml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <lccdd>
+  <info author="@billlee77">
+
```
Comment (wdconinc) [inline ip6/far_forward/B0_preshower.xml:4]: Well, actually... it should be @ajentsch

Comment (wdconinc) [reply ip6/far_forward/B0_preshower.xml:4]:
```suggestion
  <info author="@ajentsch">

```

Comment (ajentsch) [reply ip6/far_forward/B0_preshower.xml]: @wdconinc I charge in nBeers/hr.

**Comment by wdconinc** [comment]
I'm going to close this without merge and delete the branch.

**Pull request closed without merging**